data/transition-sites/walesoffice_cymru.yml

```diff
@@ -1,9 +1,9 @@
 ---
 site: walesoffice_cymru
-whitehall_slug: wales-office
+whitehall_slug: office-of-the-secretary-of-state-for-wales
 host: www.swyddfa.cymru.gov.uk
 tna_timestamp: 20130129084344
 homepage_title: Swyddfa Cymru
-homepage_furl: www.gov.uk/wales-office
-homepage: https://www.gov.uk/government/organisations/wales-office.cy
-css: wales-office
+homepage_furl: www.gov.uk/office-of-the-secretary-of-state-for-wales
```

Comment:
This should be www.gov.uk/cymru .  (You can see friendly URLs in https://short-url-manager.integration.publishing.service.gov.uk/list_short_urls )